README.md

```diff
@@ -49,6 +49,16 @@ test('full import tree mocks —third param', async () => {
   assert.equal(getFile(), 'returned to 🌲 every caller in the tree')
 })
 
+import type * as utilsType from '../src/utils.js/';
+test('allows you to specify the type of returned exports', async () => {
+  const { multiplyNumbers } = await esmock<typeof utilsType>(
```

Comment:
`esmock<typeof import('../src/utils.js')>(...)` - it's shorter and cleaner

Reply:
It's certainly shorter, not so sure about cleaner tho.